lib/datapacksexpand.js

```diff
@@ -425,6 +425,17 @@ DataPacksExpand.prototype.getNestedDataPackReferences = function(currentData, pa
                 }
                 
             } else {
+                if(!(jobInfo.maxDepth === 0) || jobInfo.includeSalesforceMetadata){
+                    let referenceInfo =  this.vlocity.datapacksutils.handleDataPackEvent('onAutoExport', referenceType, currentData);
+                    if (!jobInfo.specificManifestKeys){
+                        jobInfo.specificManifestKeys = [];
+                    } 
+                    referenceInfo.then(result => {
```

Comment:
better to use `await` on line 429 to make it more clear

Reply:
Changed to await.